src/components/modebar/manage.js

```diff
@@ -178,6 +173,32 @@ function areAllAxesFixed(fullLayout) {
     return allFixed;
 }
 
+// look for traces that support selection
+// to be updated as we add more selectPoints handlers
+function isSelectable(fullData) {
+    var selectable = false;
+
+    for(var i = 0; i < fullData.length; i++) {
+        if(selectable) break;
+
+        var trace = fullData[i];
+
+        if(!trace._module || !trace._module.selectPoints) continue;
```

Comment:
If we get past this point, all the code below has no purpose - `selectable` will always be set to true and `true` will be returned. 

Can we simplify this:

``` js
function isSelectable(fullData) {
    var selectable = false;

    for(var i = 0; i < fullData.length; i++) {
        var trace = fullData[i];
        if(!trace._module || !trace._module.selectPoints) continue
        else return true;
    }
}
```

As far as I can see this will do the same thing.